themes/mms-cloud/page.html

```diff
@@ -42,7 +42,33 @@
 
 {%- block onboarding_modal -%}{%- endblock -%}
 
-{%- block alertbar -%}{%- endblock -%}
+{%- block alertbar -%}
+    {%- if theme_has_mdb_agent %}
+        <div class="alert alert-info">
```

Comment:
Can we add a left margin of 20px to the alert info css so it lines up with the rest of the content on the page?

```
.alert-info {
    margin-bottom: 40px;
    margin-left: 20px;
}
```

Reply:
Agreed. Fixed.